app/post/post_routes.py

```diff
@@ -27,29 +28,25 @@ def manage_post(id=None, username=None):
                 user_id = data.get('user_id', None)
                 post = Post(user_id=user_id,title=title, body=body)
             elif request.method == 'PUT':
-                try:
```

Comment:
id is reserved word in python always avoid to use it Also here you are just assigning a value by reference you don't have to try